stacks/level1a_stack.py

```diff
@@ -16,8 +17,9 @@ def __init__(
         rac_bucket_name: str,
         platform_bucket_name: str,
         output_bucket_name: str,
-        lambda_schedule: Schedule = Schedule.rate(Duration.hours(12)),
         lambda_timeout: Duration = Duration.seconds(300),
+        queue_retention_period: Duration = Duration.days(14),
+        queue_visibility_timeout: Duration = Duration.hours(12),
```

Comment:
I think this is probably a good default to start with, but it might be a good idea to investigate what the actual expected lag is.